webapp/graphite/render/evaluator.py

```diff
@@ -50,11 +56,16 @@ def evaluateTokens(requestContext, tokens, replacements=None):
       # as tokens.template. this generally happens if you try to pass non-numeric/string args
       raise ValueError("invalid template() syntax, only string/numeric arguments are allowed")
 
-    func = SeriesFunctions[tokens.call.funcname]
+    try:
+      func = SeriesFunctions[tokens.call.funcname]
+    except KeyError as err:
```

Comment:
why are we catching only keyerror here? do we need to catch any other exception?

Reply:
SeriesFunctions is a dictionary with graphite function name in string as key and graphite function name as value. The only exception we know of(tried calling with invalid function name) is KeyError.